source/jormungandr/jormungandr/parking_space_availability/bss/forseti.py

```diff
@@ -0,0 +1,148 @@
+# coding: utf-8
+
+# Copyright (c) 2001-2022, Hove and/or its affiliates. All rights reserved.
+#
+# This file is part of Navitia,
+#     the software to build cool stuff with public transport.
+#
+# Hope you'll enjoy and contribute to this project,
+#     powered by Hove (www.hove.com).
+# Help us simplify mobility and open public transport:
+#     a non ending quest to the responsive locomotion way of traveling!
+#
+# LICENCE: This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+# Stay tuned using
+# twitter @navitia
+# channel `#navitia` on riot https://riot.im/app/#/room/#navitia:matrix.org
+# https://groups.google.com/d/forum/navitia
+# www.navitia.io
+from __future__ import absolute_import, print_function, unicode_literals, division
+from jormungandr import cache, app
+import pybreaker
+import logging
+import requests as requests
+from jormungandr.ptref import FeedPublisher
+from jormungandr.parking_space_availability.bss.stands import Stands, StandsStatus
+from jormungandr.parking_space_availability.bss.common_bss_provider import CommonBssProvider, BssProxyError
+import six
+
+DEFAULT_FORSETI_FEED_PUBLISHER = {
+    'id': 'forseti',
+    'name': 'forseti',
+    'license': 'Private',
+    'url': 'www.forseti.fr',
+}
+
+
+class ForsetiProvider(CommonBssProvider):
+    """
+    class managing calls to Forseti external service providing real-time BSS stands availability
+
+    """
+
+    def __init__(
+        self,
+        service_url,
+        distance=50,
+        organizations=[],
+        feed_publisher=DEFAULT_FORSETI_FEED_PUBLISHER,
+        timeout=2,
+        **kwargs
+    ):
+        self.logger = logging.getLogger(__name__)
+        self.service_url = service_url
+        self.distance = distance
+        self.timeout = timeout
+        self.network = "Forseti"
+        self.breaker = pybreaker.CircuitBreaker(
+            fail_max=kwargs.get('circuit_breaker_max_fail', app.config['CIRCUIT_BREAKER_MAX_FORSETI_FAIL']),
+            reset_timeout=kwargs.get(
+                'circuit_breaker_reset_timeout', app.config['CIRCUIT_BREAKER_FORSETI_TIMEOUT_S']
+            ),
+        )
+
+        self._feed_publisher = FeedPublisher(**feed_publisher) if feed_publisher else None
+        if not isinstance(organizations, list):
+            self.organizations = organizations.strip('[').strip(']').split(',')
```

Comment:
you can use "jsonifier" 

```suggestion
            import json
            self.organizations = json.loads(str(rganizations))
```

Reply:
:

Reply:
:+1: